src/status_im2/contexts/chat/lightbox/view.cljs

```diff
@@ -86,160 +34,88 @@
 (defn image
   [message index _ {:keys [screen-width screen-height] :as args}]
   [rn/view
-   {:style (style/image (+ screen-width seperator-width) screen-height)}
+   {:style (style/image (+ screen-width constants/seperator-width) screen-height)}
    [zoomable-image/zoomable-image message index args
-    #(toggle-opacity index args %)]
-   [rn/view {:style {:width seperator-width}}]])
+    #(utils/toggle-opacity index args %)]
+   [rn/view {:style {:width constants/seperator-width}}]])
+
+(defn lightbox-content
+  [props {:keys [data transparent? scroll-index set-full-height?]} animations derived messages index
+   callback]
+  (let [insets           (safe-area/get-insets)
+        window           (rn/get-window)
+        window-width     (:width window)
+        window-height    (:height window)
+        window-height    (if platform/android?
+                           (+ window-height (:top insets))
+                           window-height)
```

Comment:
Just another small comment.
I understand why `window-height` is being shadowed, I'd suggest to almost never do it, it's like working with mutable vars (since a var declared changes its value across the function). In this case is very simple, but I think we can change it to:

```clojure
window-height    (if platform/android?
                   (+ (:height window) (:top insets))
                   (:height window))
```